### ISSUE TITLE
Nuki 3.0 pro

### ISSUE THREAD

**Issue by TimoR1014** [mabunixda/node-red-contrib-nuki#39]
Hello

do the nodes also support the new Nuki 3.0 pro?

**Comment by jayfizzle** [comment]
or would it be possible to add the support of the new nuki 3.0 pro (wifi integrated - without bridge)

**Comment by mabunixda** [comment]
yes that would be possible - but I do not own the hardware for testing the integration

**Comment by lowrisk75** [comment]
Can we use the online API for now instead (for the Pro?) then what data would you fill up for the bridge field (hostname/ip) ? -- thank you
